aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -150,12 +148,12 @@ module aptos_framework::block {
     /// reconfiguration event.
     fun emit_genesis_block_event(vm: signer) acquires BlockResource {
         let block_metadata_ref = borrow_global_mut<BlockResource>(@aptos_framework);
-        let genesis_id = crypto_hash::new_hash_value(0 ,0);
+        let genesis_id = @0x0;
```

Comment:
What does @ do?

Reply:
magic, it's the primitive to create an address